Moderation-Policy.md

```diff
@@ -191,10 +191,10 @@ of the [Code of Conduct][].
 * Moderation of Posts authored by non-Collaborators may result in those
   non-Collaborators being Blocked temporarily or indefinitely from further
   participation in the Node.js GitHub organization.
-* In the case where a GitHub Account appears to have been created with no
-  intention to collaborate in good faith, swift actions may be taken without
-  following the above procedures including: removing Posts, Blocking
-  indefinitely, and reporting accounts to GitHub.
+* In the case where a GitHub Account, Issue, or Pull-Request appears to have been
+  created with no intention to collaborate in good faith, swift actions may be taken
```

Comment:
```suggestion
created with no intention to collaborate in good faith, actions may be taken
```
Nit: remove `swift`. For the cases where intent is not obvious, some thought and discussion might be optimal. Since we are expanding the available (irreversible) actions to take, this get a little bit more pronounced.

Reply:
Perhaps this can be discussed in a separate PR?

Reply:
+1, I'd prefer to debate the specific language in a separate PR